Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:latest as build-dev
+
+WORKDIR /biscuit
+
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -o ~/biscuit *.go
+
+FROM alpine
+
+COPY --from=build-dev /root/biscuit /usr/local/bin/.
+
+RUN apk update && apk add --no-cache \
+  ca-certificates \
+  bash \
+  openssl
+
+RUN addgroup -S appgroup && adduser -S appuser -G appgroup
+
+USER appuser
+
+ENTRYPOINT ["biscuit"]
```

Comment:
this docker file is similar to the one used by GoReleaser in https://github.com/dcoker/biscuit/pull/15 but installs `bash` as well as we're using `bash` for the test. This can be a later optimization too.